packages/bitbucket-v8-coverage-report/README.md

```diff
@@ -0,0 +1,47 @@
+# Zoosh bitbucket-v8-coverage-report
```

Comment:
Can we include that the BITBUCKET_COMMIT and BITBUCKET_REPO_FULL_NAME env variables from https://support.atlassian.com/bitbucket-cloud/docs/variables-and-secrets/ are required, if you use docker in the pipelines you have to pass those to the image